.gitignore

```diff
@@ -10,3 +10,6 @@ save-cloud-charts/save-cloud/charts
 *.log
 *.jar
 /save-backend/files/testSuites/CQFN.org/Standard+test+suites/*/*.zip
+
+# JPA Buddy
```

Comment:
you use JPA Buddy?

Reply:
Well, yes &mdash; I decided to give it a try.